### ISSUE TITLE
Error on invalid JSON

### ISSUE THREAD

**Issue by ankushdharkar** [Real-Dev-Squad/website-members#71]
We should not break the entire page, just because one json entry was incorrect. Please provide this error-handling

![image](https://user-images.githubusercontent.com/1935403/98451864-da3aca00-20fe-11eb-9ba7-a23610889407.png)

**Comment by ankurnarkhede** [comment]
@ankushdharkar , I agree that we should not show the JS error message on the screen and ideally show a server error page. Is this what is expected?

**Comment by ankushdharkar** [comment]
@ankurnarkhede We should ensure that one bad json file doesn't stop other members from showing up. Currently, if your json file has a comma missing, no member shows up on the members page
We should isolate the error on the page to that user and still show the rest of the correct users
